plugins/embed-optimizer/load.php

```diff
@@ -62,6 +56,15 @@ static function ( string $global_var_name, string $version, Closure $load ): voi
 			$GLOBALS[ $global_var_name ]['version'] = $version;
 			$GLOBALS[ $global_var_name ]['load']    = $load;
 		}
+
+		// Wait until after the plugins have loaded and the theme has loaded. The after_setup_theme action is used
+		// because it is the first action that fires once the theme is loaded. Otherwise, if the after_setup_theme
+		// action has already fired, then bootstrap right away.
+		if ( 0 === did_action( 'after_setup_theme' ) ) {
+			add_action( 'after_setup_theme', $bootstrap, PHP_INT_MIN );
+		} else {
+			$bootstrap();
+		}
```

Comment:
I don't think this is quite right. We're adding multiple bootstrap functions. We need to only do the `add_action()` call if `$GLOBALS[ $global_var_name ]` was not defined when this function was called.

Reply:
Also, perhaps this means that the bootstrap function shouldn't do `unset( $GLOBALS[ $global_var_name ] )` because then we won't know whether the bootstrap had been previously added.